packages/taquito/test/contract/rpc-contract-provider.spec.ts

```diff
@@ -19,12 +19,15 @@ import {
   InvalidCodeParameter,
   InvalidDelegationSource,
   InvalidInitParameter,
+  InvalidStakingSource,
 } from '../../src/contract/errors';
 import { preapplyResultFrom } from './helper';
 import { OpKind, ParamsWithKind, TransferTicketParams } from '../../src/operations/types';
 import { NoopParser } from '../../src/taquito';
 import { OperationBatch } from '../../src/batch/rpc-batch-provider';
 import { PvmKind } from '@taquito/rpc';
+import { InvalidAddressError } from '@taquito/utils';
```

Comment:
Error exporting from @taquito/core best to stick to core. We export it from utils only not introduce breaking change